e2e/utils/contracts.go

```diff
@@ -0,0 +1,33 @@
+package utils
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/stretchr/testify/require"
+
+	testcontract "github.com/zeta-chain/zetacore/testutil/contracts"
+)
+
+const (
+	// ErrHashRevert is the keccak256 hash of custom error "Foo()" on reverter contract
+	ErrHashRevert = "0xbfb4ebcf"
```

Comment:
```suggestion
	ErrHashRevertFee = "0xbfb4ebcf"
```

Reply:
I think you meant adding a `Foo` prefix to `ErrHashRevert`? refactor in commit https://github.com/zeta-chain/node/pull/2726/commits/92d67f5897576fc119b93838893b0aaf9a43ec08